metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
+
+A summary of the metrics the Universal Search add-on and recommendation server will record.
+
+**Definitions:**
+
+- **URL bar** — the unified address + search bar at the top of the browser.
+- **Popup** — the popup that appears when you begin typing into the URL bar.
+- **Search** — the full lifecycle, beginning when the user types a key into the url bar, and ending when the user either chooses an item from or otherwise dismisses the popup.
+
+
+## Client metrics
+
+Metrics gathered from the Universal Search add-on will be reported to the Test Pilot add-on, which will augment that information and relay it to Firefox's telemetry system via `submitExternalPing`. This is not currently implemented on the Test Pilot side; those efforts are encompassed by [issue #234](https://github.com/mozilla/testpilot/issues/234).
+
+
+### Events
+
+Universal Search will record and report each of the following events:
+
+- User selects a recommendation.
```

Comment:
You have these broken down by event, but you have a lot of duplication (eg. time from start to end).  Wouldn't "event" just be a type identifier?  So you'd record something like:

```
us.totaltime
us.querylength
us.eventtype
us.enhancers (if applicable)
us.datatype (if applicable)
us.levenstein (if applicable)
```
and then when you visualize you'd break it down into whatever events you wanted?  I know this is just the same thing you're saying here, but I'm trying to picture this from the POV of recording the data.  (that will help build the right structures to record the data)